main.py

```diff
@@ -84,11 +84,20 @@ def img(soup):
 
 def GetPage(path):
```

Comment:
Same as above, we should follow PEP8 coding style convention to name function name in lowercase, with words separated by underscores. [Here is the ref](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names)